packages/ng/statusBadge/statusBadge.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, inject, Input, OnChanges, ViewEncapsulation } from '@angular/core';
+import { LuClass, Palette } from '@lucca-front/ng/core';
+
+@Component({
+	selector: 'lu-status-badge',
+	standalone: true,
+	templateUrl: './statusBadge.component.html',
+	styleUrls: ['./statusBadge.component.scss'],
+	encapsulation: ViewEncapsulation.None,
+	providers: [LuClass],
+	host: {
+		class: 'statusBadge',
+	},
+})
+export class StatusBadgeComponent implements OnChanges {
+	#luClass = inject(LuClass);
+
+	@Input({ required: true })
+	label: string;
+
+	@Input()
+	size: 'M' | 'L' = 'M';
```

Comment:
Setting size to default `M` means that parent components won't be able to change that using just CSS, with the fact that we're checking for the value itself in `ngOnChanges`, I wouldn't set a default size here.